functions/influxdb+grafana.bats

```diff
@@ -7,16 +7,18 @@ load helpers
   run influxdb_install "Password1234"
   [ "$status" -eq 0 ]
   echo -e "# \e[32mInfluxDB installation successful." >&3
-  run systemctl is-active --quiet influxdb.service
-  [ "$status" -eq 0 ]
-  echo -e "# \e[32mInfluxDB service running." >&3
+# it does not work to test for running service at this stage 
```

Comment:
Will not be merged.

Reply:
I'm not sure why  they fail but I don't think it is because of ARM/x86. systemctl even is not a binary any more. InfluxDB isntallation for example works but the subsequent test against systemd does not.
Ok, I returned to your original test code.
We need to revisit this when build system changes are merged.